tests/tests/server_test.rs

```diff
@@ -280,8 +280,7 @@ fn basic_pipeline() {
     let events = vec![
```

Comment:
Have any tests been added that use this new feature?

Reply:
I reworked the existing test `execute_pipeline`. The test called two programs in two distinct executing program requests, but now calls an executing pipeline request. The pipeline has been statically defined in the policy.